client/src/modules/roles/modal/roleActions.html

```diff
@@ -0,0 +1,33 @@
+<div class="modal-header">
+    <ol class="headercrumb">
+      <li class="static" translate>FORM.BUTTONS.ACTIONS</li>
+      <li class="title">
+        <span translate>{{RoleActionsCtrl.role.label}}</span>
```

Comment:
Does this need to be translated?  Or is it a user-provided parameter?

Reply:
Yes it should be translated, an action should be described and every user have to understand what it means